### PR TITLE
[ZEPPELIN-3038] Network visualization not show "source" and "target" node/edge properties

### PR DESCRIPTION
**Pull request by conker84**
### What is this PR for?
The Network visualization not show "source" and "target" node/edge properties when the graph is flattened to create the table representation.

### What type of PR is it?
[Bug Fix]

### Todos
* [x] - Fixed

### Screenshot
Before:
zeppelin before

After:
zeppelin after

### What is the Jira issue?
[ZEPPELIN-3038](https://issues.apache.org/jira/browse/ZEPPELIN-3038)

### How should this be tested?
Please use this [notebook](https://gist.github.com/conker84/9574127c2389d08164423894aa93b67f)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-web/src/app/tabledata/networkdata.js [hunk 1]
```diff
@@ -40,42 +40,42 @@ export default class NetworkData extends TableData {
...
-        let name = columnNames[j].name
-        let value = name in entity && internalFieldsToJump.indexOf(name) === -1
+        const name = columnNames[j].name
+        const value = name in entity && internalFieldsToJump.indexOf(name) === -1
             ? entity[name] : entity.data[name]
-        let parsedValue = value === null || value === undefined ? '' : value
+        const parsedValue = value === null || value === undefined ? '' : value
         col.push(parsedValue)
-        col2.push({key: name, value: parsedValue})
+        // col2.push({key: name, value: parsedValue})
```
Comment (felixcheung) [inline zeppelin-web/src/app/tabledata/networkdata.js]: removed?

File: zeppelin-web/src/app/tabledata/networkdata.js [hunk 2]
```diff
@@ -40,42 +40,42 @@ export default class NetworkData extends TableData {
...
-    }))
+    })
+    const columnNames = baseColumnNames.concat(entityColumnNames)
     for (let i = 0; i < entities.length; i++) {
-      let entity = entities[i]
-      let col = []
-      let col2 = []
+      const entity = entities[i]
+      const col = []
+      // const col2 = []
```
Comment (felixcheung) [inline zeppelin-web/src/app/tabledata/networkdata.js]: removed?

**Comment by conker84** [comment]
@felixcheung I updated the description with the screenshots

**Comment by felixcheung** [comment]
ah thanks, that seems useful... @1ambda would you like to take a look?

**Comment by conker84** [comment]
Hi guys, any update?

**Comment by conker84** [comment]
No, just because i don't see tests for the visualization files. I can create it if you want!

**Comment by conker84** [comment]
Hi guys no news?

**Comment by conker84** [comment]
@felixcheung ping :)

**Comment by conker84** [comment]
hi guys, any news?

**Comment by felixcheung** [comment]
sorry - will check within the next day

**Comment by felixcheung** [comment]
merged, thanks

**Pull request closed without merging**